### PR TITLE
Updates for make install

### PR DESCRIPTION
**Pull request by mppf**
* Upgrades to 1.15
* Includes make install patch
* Uses configure, make install
* Changes to pruning third-party src directories but not Makefiles
* Includes highlight copyright info in copyright file

Reviewed by @ben-albrecht - thanks!

### CODE REVIEW

**Comment by ben-albrecht** [comment]
Could you post the lintian logs you get for a package built with this branch?

**Comment by mppf** [comment]
changes-lint.log
```
N: This typo is part of the source code and will not be fixed upstream:
O: chapel-minimal: spelling-error-in-binary usr/bin/chpl dont don't
I: chapel-minimal: arch-dep-package-has-big-usr-share 5144kB 54%
```

deb-lint.log
```
N: This typo is part of the source code and will not be fixed upstream:
O: chapel-minimal: spelling-error-in-binary usr/bin/chpl dont don't
I: chapel-minimal: arch-dep-package-has-big-usr-share 5144kB 54%
```

dsc-lint.log is empty.

**Comment by ben-albrecht** [comment]
>deb-lint.log
```
N: This typo is part of the source code and will not be fixed upstream:
O: chapel-minimal: spelling-error-in-binary usr/lib/chapel/bin/linux64/chpl funciton function
```

When making the [`chapel-minimal.lintian-overrides`](https://github.com/chapel-lang/chapel-packaging/blob/master/debian/sid/debian/chapel-minimal.lintian-overrides) file, I couldn't figure out how to make the override specific to the error message contents, so the override was applying to all of typo errors.

We probably want to patch that new typo, and optionally figure out how to make our lintian override specific to the `dont -> don't` case.

For reference, [Here's the section discussing lintian overrides](https://lintian.debian.org/manual/section-2.4.html) in the manual.

File: debian/sid/debian/patches/funciton.patch [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Description: Fix funciton typo
+Author: Ben Albrecht <balbrecht@cray.com>
+Origin: backport, https://github.com/chapel-lang/chapel/pull/5964
```
Comment (ben-albrecht) [inline debian/sid/debian/patches/funciton.patch]: We may want to include the [forwarded](http://dep.debian.net/deps/dep3/) field to make it clear that this patch has been made upstream.

Comment (mppf) [inline debian/sid/debian/patches/funciton.patch:3]: backport means we got it from upstream

**Review by ben-albrecht — APPROVED** [review]
Looks good.

Since we're trying to reflect chapel-lang/chapel practices in this repository, would you mind adding a brief summary of the major changes added in the PR msg?

**Pull request merged**